File: BE/src/main/java/com/ssafy/e106/domain/subscriptionusage/dto/response/SubscriptionUsageDailyPointResponse.java
package com.ssafy.e106.domain.subscriptionusage.dto.response;

import java.time.LocalDate;

import com.ssafy.e106.domain.subscription.enums.ServiceCategory;

public record SubscriptionUsageDailyPointResponse(
    LocalDate usageDate,
    Integer totalUsedMinutes,
    Integer ottUsedMinutes,
    Integer musicUsedMinutes,
    Integer aiUsedMinutes,
    ServiceCategory dominantCategory) {
}

File: FE/app/src/main/java/com/ssafy/e106/data/dto/subscriptionusage/SubscriptionUsageDailyPointResponse.kt
package com.ssafy.e106.data.dto.subscriptionusage

import kotlinx.serialization.Serializable

@Serializable
data class SubscriptionUsageDailyPointResponse(
    val usageDate: String,
    val totalUsedMinutes: Int,
    val ottUsedMinutes: Int,
    val musicUsedMinutes: Int,
    val aiUsedMinutes: Int,
    val dominantCategory: String? = null,
)
